internal/users/testdata/inactive_users_in_many_groups.group

```diff
@@ -0,0 +1,6 @@
+localgroup1:x:41:inactiveuser,inactiveuser2,inactiveuser3
```

Comment:
For that fixture, I would mix the use case to not have the 3 same users in the 3 groups.